vignettes/explorer.Rmd

```diff
@@ -0,0 +1,98 @@
+---
+title: "Summary and Usage of ExploreR"
+author: "Vignette Author"
+date: "`r Sys.Date()`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Vignette Title}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+Welcome to explorR This is a small package that will provide very useful summaries of your data frame. It is best used before youer Exploratory Data Analysis. The package is composed of three functions. The instrunctions below will explain the use and the specfifc output based on the data_frame input provided for the function.
```

Comment:
Just a few punctuation and spelling corrections needed.
- period after exploreR
- spelling: explorR
- spelling: youer
- spelling: instrunctions
- spelling: specfifc
- spelling: data_frame